sample-input/simple-lattice/simple-lattice.py

```diff
@@ -24,18 +24,13 @@
 
 openmoc.log.py_printf('NORMAL', 'Creating surfaces...')
 
-left = openmoc.XPlane(x=-2.0, name='left')
-right = openmoc.XPlane(x=2.0, name='right')
-bottom = openmoc.YPlane(y=-2.0, name='bottom')
-top = openmoc.YPlane(y=2.0, name='top')
-boundaries = [left, right, bottom, top]
+boundary = openmoc.RectangularPrism(4., 4.)
+boundary.setBoundaryType(openmoc.PERIODIC)
```

Comment:
Does this mean we would no longer support bounding boxes with different boundary types?